python/test/test_polars.py

```diff
@@ -73,10 +80,55 @@ def test_lazyframe_ffi():
     t_ident = (lf_domain, dp.symmetric_distance()) >> dp.t.then_identity()
     assert t_ident(lf).collect().equals(lf.collect())
 
+
 def test_expr_ffi():
     """ensure that lazyframes can be passed to/from Rust"""
     pl = pytest.importorskip("polars")
     lf_domain, lf = example_lf()
     expr_domain = dp.expr_domain(lf_domain, "select")
     t_ident = (expr_domain, dp.symmetric_distance()) >> dp.t.then_identity()
     assert str(t_ident((lf, pl.col("A")))[1]) == str(pl.col("A"))
+
+
+def test_private_lazyframe():
+    pl = pytest.importorskip("polars")
+    lf_domain, lf = example_lf()
+
+    with pytest.raises(dp.OpenDPException):
```

Comment:
Presumably it's possible to call `make_private_lazyframe` and not get an error -- Also test that?

Consider naming the arguments: With more than 3 positional arguments, I worry about about things getting out of order. (Seems like there should be a lint check for this.)

And if there's a message than would be useful to the end user, test that, to make sure we don't start throwing a new, more cryptic error, at some point.

Reply:
At this point in the PR stack, since the only functionality implemented is the domains/metrics and this constructor entry point, it is not possible to call it and get a success. Later PRs extend these tests to check for success.